annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/index/Fix.java

```diff
@@ -54,10 +54,13 @@ public class Fix {
    */
   public boolean fixSourceIsInTarget;
 
+  public int count;
```

Comment:
What is this field for?  Javadoc?  I'd prefer accessor methods to a public, non-final field (and maybe `fixSourceIsInTarget` should be final?)

Reply:
Sorry, That field is a left over from previous design in PR #106 which is closed.  Removed it entirely.